packages/addons/service/net-snmp/source/bin/snmpd.start

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+. /etc/profile
+
+# service setup
+# service.net-snmp
+
+ADDON_ID=service.net-snmp
+ADDON_DIR="$HOME/.kodi/addons/$ADDON_ID"
+ADDON_HOME="$HOME/.kodi/userdata/addon_data/$ADDON_ID"
+
+chmod a+x $ADDON_DIR/bin/*
```

Comment:
oe_setup_addon does chmod a+x $ADDON_DIR/bin and adds it to PATH
Move binaries from sbin to bin (unless you want to prevent binaries in sbin from being in PATH)